tests/integration/test_wazuh_db/test_agent_database_version.py

```diff
@@ -0,0 +1,125 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: Wazuh-db is the daemon in charge of the databases with all the Wazuh persistent information, exposing a socket
+       to receive requests and provide information. The Wazuh core uses list-based databases to store information
+       related to agent keys, and FIM/Rootcheck event data.
+       This test checks that the agent database version is the expected one. To do this, it performs a query to the agent
+       database that gets the database version.
+
+tier: 0
+
+modules:
+    - wazuh_db
+
+components:
+    - manager
+
+daemons:
+    - wazuh-db
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - CentOS 6
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Ubuntu Xenial
+    - Ubuntu Trusty
+    - Debian Buster
+    - Debian Stretch
+    - Debian Jessie
+    - Debian Wheezy
+    - Red Hat 8
+    - Red Hat 7
+    - Red Hat 6
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/reference/daemons/wazuh-db.html
+
+tags:
+    - wazuh_db
+'''
+import pytest
+
+from wazuh_testing.constants.executions import TIER0, SERVER, LINUX
+from wazuh_testing.utils.database import query_wdb
+from wazuh_testing.tools.simulators import agent_simulator as ag
+from wazuh_testing.utils.manage_agents import remove_all_agents
+
+
+# Marks
+pytestmark = [TIER0, LINUX, SERVER]
+
+# Variables
+expected_database_version = '12'
```

Comment:
What if this changes?

Reply:
As far as I know, they have been updating this value when it changes: https://github.com/wazuh/wazuh-qa/commits/master/tests/integration/test_wazuh_db/test_agent_database_version.py

Reply:
This is difficult to maintain. Do you think we can do this differently?

Reply:
I tried to get the version from the logs, but it does not seem to be printed unless it gets upgraded. I don't know if there is any other way to obtain it besides from the database itself.